kotest-assertions/kotest-assertions-json/src/jvmTest/kotlin/com/sksamuel/kotest/tests/json/EqualTest.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("DEPRECATION") // Remove when removing shouldEqualJson legacy options
```

Comment:
I'd suggest not doing file-level suppress. I know that suppressing every element is more verbose, but with a file-level suppress I think there's more of a chance that it'll be missed and not removed, and it might cause confusion later. But you added a comment, which I think is a good idea, so that will help.

I'll leave it up to you.

Reply:
I thought it would be required to get rid of the warnings from imports, but perhaps they don't cause warnings. I'll check how spammy it would be to get rid of.

Reply:
Ah true, I didn't consider imports. In which case, a file-level suppress makes sense.